developer-docs-site/docs/releases/devnet-release.md

```diff
@@ -13,6 +13,10 @@ See the [Latest Aptos Releases](./index.md) index for how to subscribe to GitHub
 
 [Aptos Framework Release v1.2.0](https://github.com/aptos-labs/aptos-core/releases/tag/aptos-framework-v1.2.0)
```

Comment:
Is devnet on v1.3 yet? @perryjrandall

Reply:
Yes, it has been for a while

Reply:
Although we dont technically assign a version, so a hash is more correct here?

Reply:
From the internal sheet, using:
`1d0dc2c75da1e397effc8bcfa14114652462654d`

Reply:
Please take another look.